scripts/run-e2e.ts

```diff
@@ -127,7 +127,7 @@ const initStorybook = async ({ cwd, autoDetect = true, name }: Options) => {
     const type = autoDetect ? '' : `--type ${name}`;
 
     const sbCLICommand = useLocalSbCli
-      ? 'node ../../storybook/lib/cli/dist/generate'
+      ? 'node ../../storybook/lib/cli/dist/esm/generate'
```

Comment:
I would have used `cjs` to ensure compatibility with Node10+ but I'm not sure about that.

🤔 I dived into `lib/cli/dist/esm` and it looks there are `require('...')`, is it normal @ndelangen? I thought there would be only `import` statements.

Reply:
Tested locally with Node 10.x, everything was ok.